ClimateTRACE/run/gcp_utils.py

```diff
@@ -74,19 +72,45 @@ def machine_compute_resource(*, machine_type: str, region: str, project_id: str)
     return cpu_milli, memory_mib
 
 
+def wait_for_jobs_to_succeed(batch_jobs: Sequence[batch.Job]):
```

Comment:
Can you explain the logic behind this?

Reply:
it was just moving stuff around which already existed. git thinks its new stuff but it isn't. We were never using this function but I kept it there anyway.